src/Apps/erp/modules/home/styles.css

```
@@ -0,0 +1,92 @@
+  --accent: #4cc9f0;
+}
+
+.quiz-root {
+  min-height: 100vh;
+  color: var(--text);
+  background:
+    radial-gradient(circle at 20% 10%, #19386a 0%, transparent 30%),
+    radial-gradient(circle at 80% 80%, #1c406f 0%, transparent 35%),
+    linear-gradient(180deg, #050c19 0%, #071022 100%);
+  padding: 1.2rem;
+  font-family: Inter, Segoe UI, Roboto, sans-serif;
```

Stylelint (17.9.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)